Author: Pino Toscano <pino@kde.org>
Description: reduce overlinking when possible
Applied-Upstream: 4.7.1, commit:9730f0c39fb35f7813f5972540054fd14a682731
Last-Update: 2011-08-11

--- a/qtscript/CMakeLists.txt
+++ b/qtscript/CMakeLists.txt
@@ -23,7 +23,7 @@
 ENDIF(CMAKE_CXX_FLAGS MATCHES "-fvisibility")
 add_library(smokeqtscript SHARED ${smokeqtscript_LIB_SRCS})
 
-target_link_libraries(smokeqtscript smokeqtcore ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTSCRIPT_LIBRARY})
+target_link_libraries(smokeqtscript smokeqtcore ${QT_QTCORE_LIBRARY} ${QT_QTSCRIPT_LIBRARY})
 
 set_target_properties(smokeqtscript PROPERTIES VERSION 3.0.0 SOVERSION 3 )
 
--- a/qtwebkit/CMakeLists.txt
+++ b/qtwebkit/CMakeLists.txt
@@ -27,7 +27,6 @@
                         smokeqtgui 
                         ${QT_QTCORE_LIBRARY} 
                         ${QT_QTGUI_LIBRARY} 
-                        ${QT_QTNETWORK_LIBRARY} 
                         ${QT_QTWEBKIT_LIBRARY} )
 
 set_target_properties(smokeqtwebkit PROPERTIES VERSION 3.0.0 SOVERSION 3 )
